portfolio/src/main/webapp/script.js

```diff
@@ -13,10 +13,18 @@
 // limitations under the License.
 
 /**
- * Adds "Hello Naa'Oli!" to the page.
+ * Adds greetings to the page.
  */
-function getHelloName() {
-
-  const helloNameContainer = document.getElementById('container');
-  helloNameContainer.innerText = "Hello Naa'Oli!";
-}
+async function getHelloName() {
```

Comment:
since you moved the event listener in here, you should probably rename this to be `load()` or something instead of `getHelloName`, and update the documentation of the function accordingly.